transformer_engine/common/recipe.py

```diff
@@ -98,6 +98,9 @@ def scaling_factor_compute(amax: Tensor,
     override_linear_precision: Tuple(bool, bool, bool), default=(False, False, False)
                               Whether or not the execute the `fprop`, `dgrad`, and `wgrad`
                               GEMMs (respectively) in higher precision when using FP8.
+    reduce_amax: bool, default = `True`
+                If set to `True`, the `amax` value for fp8 tensors is reduced across the given
+                `fp8_group` using the `fp8_autocast` API after every iteration.
```

Comment:
This is unclear to me - what is `fp8_autocast`'s role in this sentence?

Reply:
It is to mention that the global amax reduction happens across the `fp8_group` argument, which is specified via the `fp8_autocast` API